pyccel/parser/semantic.py

```diff
@@ -759,6 +759,14 @@ def _visit_CommentBlock(self, expr, **settings):
         return expr
     def _visit_AnnotatedComment(self, expr, **settings):
         return expr
+    def _visit_OMP_ForLoop(self, expr, **settings):
+        return expr
+    def _visit_OMP_ParallelConstruct(self, expr, **settings):
+        return expr
+    def _visit_OMP_SingleConstruct(self, expr, **settings):
+        return expr
+    def _visit_Omp_EndClause(self, expr, **settings):
+        return expr
```

Comment:
Don't these classes inherit from `AnnotatedComment` ? If so then these functions are not necessary as pyccel will fall back to `_visit_AnnotatedComment` which does the same thing